internal/cmd/genop/genop.go

```diff
@@ -63,23 +63,73 @@ func {{$name}}(n *node) {
 			{{else}}
 			v1 := genValueInt(c1)
 			{{end -}}
+			{{- if (eq $name "add")}}
+			if n.typ.cat != interfaceT || len(n.typ.field) > 0 {
+				n.exec = func(f *frame) bltn {
+					_, j := v1(f)
+					dest(f).SetInt(i + j)
+					return next
+				}
+				return
+			}
+			var valf func(sum int64) reflect.Value
+			switch typ.Kind() {
```

Comment:
Can you add a TODO here mentioning that this has to be completed with all missing types (int8, ....) ?

Reply:
done